modules/spo-uninstall-console.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * security/security_profiles_operator/spo-troubleshooting.adoc
+
+:_content-type: PROCEDURE
+[id="spo-uninstall-console_{context}"]
+= Uninstall the Security Profiles Operator using the web console
+
+To remove the Security Profiles Operator, you must first delete the `seccomp` and SELinux profiles. After the profiles are removed, you can then remove the Operator and its namespace by deleting the *openshift-security-profiles* project.
+
+.Prerequisites
+
+* Access to an {product-title} cluster that uses an account with `cluster-admin` permissions.
+* The Security Profiles Operator is installed.
+
+.Procedure
+
+To remove the Security Profiles Operator by using the {product-title} web console:
+
+. Navigate to the *Operators* -> *Installed Operators* page.
+
+. Delete all `seccomp` profiles, SELinux profiles, and webhook configurations.
```

Comment:
Sorry, I discussed with @jhrozek again, he  thought SPO should delete the mutatingwebhookconfigurations spo-mutating-webhook-configuration  when "Uninstall Operator". Thus for webhook issue, a bug https://issues.redhat.com/browse/OCPBUGS-4687 was created to track.
For the document, I think "Delete all Seccomp profiles, SELinux profiles in all namespaces" would be more accuarate.

Reply:
That works for me, thanks @jhrozek !
@xiaojiey  please let me know if you have any additional feedback. I would like to update the PR and address all feedback by the end of the week. Thanks!